### ISSUE TITLE
Bloop should explain how to start itself when it cannot connect to the server

### ISSUE THREAD

**Issue by Duhemm** [scalacenter/bloop#209]
Currently, running `bloop` when there's no server started prints:

> Could not connect to server 127.0.0.1:8212

It would be better to add something along the lines of:

> Have you forgotten to start the server? It can be started with `blp-server`

Moreover, if we're changing the name of the server executable, that will make it harder to discover, and more users risk hitting that.

This should be improved in the python script.

**Comment by propensive** [comment]
I have wrapped `bloop` in a script which checks whether the server is running, and if not, starts it in a `screen` session, something like,
```
(netstat -ln | grep -q `:8212 ' || (screen -X -S bloop quit && screen -S bloop -d -m ~/.bloop/blp-server && sleep 1)) && bloop "$@"
```
There's a lot of ugly stuff in there, and it relies on `screen` and `netstat` and (worst of all) `sleep`, but it has meant I haven't had to think much about starter the server when it's not running.

Might be useful as the basis for something less hacky...

**Comment by olafurpg** [comment]
On OSX, I've found launch agents to work really well. I set it up once in December and the server has always been up when I've run on `bloop` since then. To restart the server I run `launchctl stop Bloop`.

**Comment by jvican** [comment]
For now, I lean towards not focusing on this issue and letting users deal with it manually. In the future, it would be nice if bloop starts the server on its own, but this is not something we should focus on in the long term.
  
The very notion of a client-server should be ingrained in people's mind when they use bloop, and I find this kind of user error to be beneficial in the long-term: it prevents users from running two servers.

**Comment by jvican** [comment]
Fixed in #293.
